entente/test_rigid_transform.py

```diff
@@ -89,6 +89,10 @@ def test_rigid_rotation_single_point():
 
 
 @pytest.mark.skipif(env_flag("CI") is True, reason="failing in CI when numpy>=1.19.3")
+# This test currently has two valid answers.  On numpy 1.19.3 and higher it returns a different
+# but still accurate value.
+# https://github.com/lace/entente/issues/195
+# TODO: Revise test so that it only has one valid answer.
```

Comment:
```suggestion
# TODO: Clarify the intent of this test.
#
# This test produces different answers on numpy 1.19.3 and higher on certain architectures. This is
# because there are duplicate singular values.
#
# Since a reflection through the origin in 3-space is not a rotation at all, and there is no rotation matrix
# which matches the effect of the reflection through origin, it's not clear what the correct result
# should be.
#
# https://github.com/numpy/numpy/issues/22945
# https://github.com/lace/entente/issues/195
```